src/scss/components/settings/_settings-panel-page-open-button.scss

```diff
@@ -4,15 +4,19 @@
 %ui-settingspanelpageopenbutton {
   @extend %ui-button;
 
-  background-image: url('../../assets/images/setting.svg');
-  max-height: .8em;
+  .#{$prefix}-icon {
+    background-image: url('../../assets/images/setting.svg');
+    max-height: .8em;
+  }
 
   &:hover {
     @include svg-icon-shadow;
   }
 
   &.#{$prefix}-on {
-    transform: rotate(30deg);
+    .#{$prefix}-icon {
+      transform: rotate(30deg);
+    }
```

Comment:
Only rotate the image and not the whole button as this would break additional styling.